### DISCUSSION TITLE
总结了一下22年用这个模板遇到的一些问题

### DISCUSSION THREAD

**Discussion by KaiWWWWW** [xen0n/JNUthesis#26] [General]
本人用的是texstudio+texlive2019
- 字体选项可以用winfonts，
- 英文摘要为两端对齐可以将cls文件的\defaultfontfeatures{Mapping=tex-text}注释，
- table环境加一个\small(保证字号比正文小一号)，
- 画子图可以用subcaption宏包，
- 参考文献如果遇到中文中出现et al，可以在bib文件内加参数language={zh}

最后，感谢作者制作的模板

**Comment by xen0n** [comment]
十分感动，2202 年了，还有活人来用这个项目，居然在近现代 texlive 上还能跑通，我都毕业五年了！ :facepalm:

这些东西改天编辑到 README 里。。。

临 `\begin{tabular}` 涕零，不知所言

**Comment by xen0n** [comment]
采访下，没人因为排版、字体问题找你茬吧？

我们当年应该把能趟的坑都趟了，很多个学院（那时候人工智能和计算机学院还叫树莓），我们的一次次重复调整、打印，只为了后人能省点A4纸钱

**Comment by KaiWWWWW** [comment]
我们这届还没开始答辩，最近刚刚写完论文，就顺便记录一下遇到的问题
如果后面遇到问题再来记录一下，顺便希望自己能混过答辩

**Comment by xen0n** [comment]
> 我们这届还没开始答辩，最近刚刚写完论文，就顺便记录一下遇到的问题
> 如果后面遇到问题再来记录一下，顺便希望自己能混过答辩

:accept:

祝顺利！

**Comment by aaa1999** [comment]
> 本人用的是texstudio+texlive2019
> 
> * 字体选项可以用winfonts，
> * 英文摘要为两端对齐可以将cls文件的\defaultfontfeatures{Mapping=tex-text}注释，
> * table环境加一个\small(保证字号比正文小一号)，
> * 画子图可以用subcaption宏包，
> * 参考文献如果遇到中文中出现et al，可以在bib文件内加参数language={zh}
> 
> 最后，感谢作者制作的模板

想知道您是怎么编译的呢，在win10平台。

**Reply by KaiWWWWW** [reply]
我是在texstudio上直接编译的
